lib/src/main/kotlin/app/cash/quiver/Outcome.kt

```diff
@@ -24,11 +24,49 @@ import kotlin.experimental.ExperimentalTypeInference
  * as [`map`](app.cash.quiver.Outcome.map), [`flatMap`](app.cash.quiver.Outcome.flatMap) and
  * [`zip`](app.cash.quiver.Outcome.zip).
  *
+ * There are three primary constructors:
+ *
+ * ```kotlin
+ * data class Present<A>(val value: A) : Outcome<Nothing, A>
+ * data class Failure<E>(val error: E) : Outcome<E, Nothing>
+ * object Absent : Outcome<Nothing, Nothing>
+ * ```
+ *
+ * or you can use the extension methods thusly:
+ *
+ * ```kotlin
+ * A.present()
+ * E.failure()
+ * ```
+ *
+ * You can also easily convert an `Either<Option<A>>` to an Outcome using `toOutcome()`
```

Comment:
nit: `toOutcome()` can link directly to the documentation page for `toOutcome()`: 
```suggestion
 * You can also easily convert an `Either<Option<A>>` to an Outcome using  [`toOutcome()`](app.cash.quiver.toOutcome)
```

Reply:
This can probably be a separate pull request to add links across all of our documentation